Vandhana

Vandhana-Mohan opened the pull request:
Added Tables user, favorite

Added user_id to reviews.

Added user controller and user query

Vandhana-Mohan committed:
PR merged

Vandhana-Mohan committed:
Merge branch 'main' into vandhana

Vandhana-Mohan committed:
add user table

Vandhana-Mohan committed:
add favorite table

Vandhana-Mohan committed:
edit errors in user table

Vandhana-Mohan committed:
add user, favorite in app.js, add new controllers, query files

Vandhana-Mohan committed:
add user controller

Vandhana-Mohan committed:
add user query

Vandhana-Mohan committed:
fix user query error

Vandhana-Mohan committed:
fix errors in user controller

DiandreMiller reviewed (changes requested):
There shouldn't be a function to get all users in the controller file, or the queries file. Also get rid of the commented out code.

Vandhana-Mohan committed:
replaced user with shopper, user is a keyword and cant be used

Vandhana-Mohan commented:
Removed get all users from controller and query

comments is temporary, i need them, 

also replaced table name user to shopper, user is a psql keyword, there was an error

Vandhana-Mohan reviewed (commented):
Removed get all users from controller and query

comments is temporary, i need them,

also replaced table name user to shopper, user is a psql keyword, there was an error
